tools/install-powershell.ps1

```diff
@@ -174,7 +174,7 @@ try {
         }
 
         Write-Verbose "Change icon to disambiguate it from a released installation" -Verbose
-        & "~/.rcedit/rcedit-x64.exe" "$Destination\pwsh.exe" --set-icon "$Destination\assets\Powershell_av_colors.ico"
+        & "~/.rcedit/rcedit-x64.exe" "$Destination\pwsh.exe" --set-icon "$Destination\assets\Powershell_avatar.ico"
```

Comment:
why isn't this one of the same icons set during the build?

Reply:
This is for the daily build only, see my summary.

Reply:
ok, makes sense.